_config/protected.yml

```diff
@@ -10,3 +10,6 @@
 - /video/youtube/i1aiUaa8RZ8/
 - /video/youtube/LbzkY9nrgdo/
 - /video/youtube/Q37niG_hCcA/
+- /video/youtube/pIURFD15WJ0/
```

Comment:
I've added this video to a playlist watched by the site. It should be available in this PR build after a rebuild. It'll be on production in < 6 hours.